src/main.rs

```diff
@@ -1,24 +1,19 @@
-extern crate chrono;
-extern crate elf;
-extern crate tar;
-#[macro_use]
-extern crate structopt;
-
+use cmdline::Opt;
```

Comment:
Using the full path is nice as it tells people right in the source where the very short and not unique "Opt" object is from.

Reply:
Fixed.